conf/messages

```diff
@@ -169,8 +169,8 @@ amls.inset.p1=You must be registered with a money laundering supervisory body be
 amls.form.supervisory_body=Name of supervisory body
 amls.form.membership_number=Membership number (optional)
 amls.hint.expandable=I am not registered with a money laundering supervisory body
-amls.hint.expandable.p1=You cannot apply for an agent services account if you are not registered.
-amls.hint.expandable.p2=For details about who can apply, <a href="{0}" target="_blank" id="guidance">check the guidance (opens in a new window or tab)</a>.
```

Comment:
won't the message for `amls.hint.expandable.p2` in app/uk/gov/hmrc/agentoverseasfrontend/views/application/anti_money_laundering.scala.html

change since it's no longer being fed a href?

Reply:
Yes, well spotted. The redundant argument was simply discarded so I didn't notice it but I have now removed it.